packages/core/src/routes/organization/roles.ts

```diff
@@ -66,6 +68,7 @@ export default function organizationRoleRoutes<T extends AuthedRouter>(
 
   router.post(
     '/',
+    koaQuotaGuard({ key: 'organizationsEnabled', quota }),
```

Comment:
if we need to guard so many routes, why don't we add a middleware to the organization root router? (`packages/core/src/routes/organization/index.ts`)

Reply:
Bummer... You are absolutely right